src/lib/tests/components/default.svelte

```diff
@@ -9,9 +9,11 @@
 	export let return_value: (value: unknown) => void = () => {};
 	export let argument = 0;
 
-	const default_task = task(fn);
+	const default_task = task.default(fn);
+	const options_task = task(fn);
```

Comment:
I wonder if how you had it originally (`task(fn, {kind: 'default'})`) makes more sense here and then we can add a specific test for "`task` uses 'default'"

Reply:
I agree....fixing this here and then i will write a new batch of tests